### ISSUE TITLE
Modal style broken?

### ISSUE THREAD

**Issue by obernardovieira** [web3connect/web3connect#31]
Hello,
I'm currently having a problem related to UI. You can see in the image below that it is broken. The component is totally empty (code is also below). Although I was able to fix it with a few CSS. I'm asking myself if I'm missing something or it is really broken.

The package is working well, I've tried to connect to metamask, and fortmatic, all cool, except the UI.
```javascript
import Web3Connect from "web3connect";
import Web3 from "web3";
import React, { Component } from 'react';

class App extends Component {
    render() {
        return (
            <div>
                <Web3Connect.Button
                    providerOptions={{
                        fortmatic: {
                            key: "my_key" // required
                        }
                    }}
                    onConnect={(provider) => {
                        const web3 = new Web3(provider); // add provider to web3
                        console.log('web3', web3);
                    }}
                    onClose={() => {
                        console.log("Web3Connect Modal Closed"); // modal has closed
                    }}
                />
            </div>
        );
    }
}

export default App;
```
![image](https://user-images.githubusercontent.com/19441097/60278351-2ebd1b00-98f7-11e9-9b9b-ee60e25a19d7.png)

**Comment by crisgarner** [comment]
Which browser are you using? Just used your code on the example file and tested in brave.

![Screenshot from 2019-06-27 12-35-53](https://user-images.githubusercontent.com/578688/60291511-3faa6400-98d8-11e9-861f-2dd16a214c9f.png)
![Screenshot from 2019-06-27 12-36-07](https://user-images.githubusercontent.com/578688/60291512-3faa6400-98d8-11e9-92f9-dbb2291e2759.png)

**Comment by obernardovieira** [comment]
Initially Vivaldi (chromium based), but after looking at your message, I've tried chrome, firefox and brave.
I don't have any style file ... :thinking:

**Comment by austintgriffith** [comment]
Yeah I’m seeding the exact same thing:
Reproduce by making a simple Create React App and pasting in the React Button:
![2019-06-30 09 29 20](https://user-images.githubusercontent.com/2653167/60398792-a10d4580-9b19-11e9-989c-53304db536e1.jpg)

**Comment by pedrouid** [comment]
Fixed on web3connect@1.0.0-beta.11
